neqo-crypto/build.rs

```diff
@@ -169,8 +169,9 @@ fn dynamic_link_both(extra_libs: &[&str]) {
 
 fn static_link(nsstarget: &PathBuf) {
     let lib_dir = nsstarget.join("lib");
+    // Include a RPATH directive to make debugging easier.
     println!(
-        "cargo:rustc-link-search=native={}",
+        "cargo:rustc-cdylib-link-arg=-Wl,-rpath,{}",
```

Comment:
Note that I don't think that this works.  But the previous line was a duplicate.  So I should just remove this.